### PR TITLE
Fixing dependency isolation tests for linux consumption

### PR DESCRIPTION
**Pull request by gavin-aguiar**
## Description
Fixing dependency isolation tests for linux consumption by removing the __init__.py file from the azure directory. This change can be removed once worker 4.0.0 is released

Fixes #

---

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azure_functions_worker/testutils_lc.py [hunk 1]
```diff
@@ -181,11 +193,15 @@ def spawn_container(self,
         run_process = subprocess.run(args=run_cmd,
                                      stdout=subprocess.PIPE,
                                      stderr=subprocess.PIPE)
+
         if run_process.returncode != 0:
             raise RuntimeError('Failed to spawn docker container for'
                                f' {image} with uuid {self._uuid}.'
                                f' stderr: {run_process.stderr}')
 
+        # Remove once worker version 4.0.0 is released
```
Comment (vrdmr) [inline azure_functions_worker/testutils_lc.py:202]: Add an issue for us to remind this.

This is released though - right?

File: azure_functions_worker/testutils_lc.py [hunk 2]
```diff
@@ -129,12 +129,24 @@ def _find_latest_mesh_image(cls,
                                f' Status {response.status_code}')
 
         tag_list = response.json().get('tags', [])
+        # Removing images with a -upgrade
```
Comment (vrdmr) [inline azure_functions_worker/testutils_lc.py:132]: Better comment to describe why we do this.

**Review by vrdmr — APPROVED** [review]

File: azure_functions_worker/testutils_lc.py (hunk 1 above)
Comment (gavin-aguiar) [reply azure_functions_worker/testutils_lc.py:202]: Its in prerelease. https://github.com/Azure/azure-functions-python-worker/releases.

Added a task for tracking.

File: azure_functions_worker/testutils_lc.py (hunk 2 above)
Comment (gavin-aguiar) [reply azure_functions_worker/testutils_lc.py]: Fixed

**Pull request merged**